app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
So I think that we publish the schemas only for non-dev versions? So we'll have the
`"id": "https://diaspora.github.io/diaspora_federation/schemas/federation_entities-0.2.0-dev.json"` to identify the version in schema now, but we'll actually upload the schema to that address only when we make release and switch from `0.2.0-dev` to `0.2.0`? So as for the repository we'll have `app/schemas/federation_entities.json` file to track the last version and we'll have `docs/schemas` folder with all the released schema versions copied there.